packages/tooltip/src/Tooltip/Tooltip.tsx

```diff
@@ -155,6 +155,12 @@ function Tooltip({
             onBlur: (e: MouseEvent) => {
               userTriggerHandler('onBlur', e);
               handleClose();
+
+              // If a user clicks on the tooltip trigger and immediately navigates to a new window/tab and then back, document.activeElement is still the trigger which will cause the
+              // tooltip to reappear even if not hovered. This blurs the last active element to prevent it from unexpectedly reappearing.
+              if (document.activeElement instanceof HTMLElement) {
```

Comment:
Should we be more explicit and check if `document.activeElement` is the trigger itself?

Reply:
good call! will put a ref on the trigger to compare against `document.activeElement` :)